.builder/actions/mock_server_setup.py

```diff
@@ -29,13 +29,12 @@ def run(self, env):
 
         # set cmake flag so mock server tests are enabled
         env.project.config['cmake_args'].append(
-            '-DENABLE_MOCK_SERVER_TESTS=ON')
+            ['-DENABLE_MOCK_SERVER_TESTS=ON'])
```

Comment:
```suggestion
            '-DENABLE_MOCK_SERVER_TESTS=ON')
```

did this break anything? 
this used to add a string to the list
now it adds a list containing a string to the list

either revert this, or use `.extend(['-DENABLE_MOCK_SERVER_TESTS=ON'])`